authnid/make_db.py

```diff
@@ -0,0 +1,27 @@
+import psycopg2
+import psycopg2.extras
+
+def connect_db(uri):
+    return psycopg2.connect(uri, cursor_factory=psycopg2.extras.DictCursor)
+
+def make_cursor(connection):
+    return connection.cursor()
+
+def make_db(config):
+    connection = connect_db(config['DATABASE_URI'])
+    cursor = make_cursor(connection)
+    cursor.execute('''
+        CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+        CREATE TABLE IF NOT EXISTS users (
+            id uuid DEFAULT uuid_generate_v4() PRIMARY KEY,
+            email VARCHAR(254),
```

Comment:
Is there an advantage to specifying the length of these, rather than just `varchar`?

Reply:
Probably not. Just trying to be specific. Do you think that's asking for trouble?

Reply:
Yeah, we don't need that level of optimization.

Reply:
Okay. Removed those and added autocommit on two new commits.